components/measurement/nettests/WebConnectivity.js

```diff
@@ -294,27 +294,23 @@ const WebConnectivityDetails = ({
   let status = 'default'
   let summaryText = ''
 
-  const reason = messages[`blockingReason.${blocking}`] && intl.formatMessage(messages[`blockingReason.${blocking}`])
+  let reason = messages[`blockingReason.${blocking}`] && intl.formatMessage(messages[`blockingReason.${blocking}`])
```

Comment:
@hellais we have come a full circle with extracting `reason` outside the `if-else` chain and back to resetting it inside the chain :joy: